app/views/course/assessment/answer/programming/_test_cases.html.slim

```diff
@@ -40,7 +40,8 @@ table.table.table-striped.table-hover
             - test_case_result = answer_test_results_hash[test_case]
             = fa_icon('check') if test_case_result && test_case_result.passed?
 
-- failed_private_test_cases = private_test_cases.select { |t| !test_case_result || !test_case_result.passed? }
+- failed_private_test_cases = private_test_cases.select {|t| !answer_test_results_hash[t] || \
```

Comment:
I don't understand what this does: why are we displaying private test cases to the student?